Mathlib/RingTheory/OreLocalization/OreSet.lean

```diff
@@ -21,6 +21,23 @@ This defines left Ore sets on arbitrary monoids.
 
 -/
 
+namespace AddOreLocalization
+
+/-- A submonoid `S` of an additive monoid `R` is (left) Ore if common summands on the right can be
+turned into common summands on the left, and if each pair of `r : R` and `s : S` admits an Ore
+minuend `v : R` and an Ore subtrahend `u : S` such that `u + r = v + s`. -/
+class OreSet {R : Type*} [AddMonoid R] (S : AddSubmonoid R) where
```

Comment:
Right now, the way to declare that something is and additive Ore set is by `open`ing  `AddOreLocalization` and then declaring `OreSet`; in analogy with `AddMonoid` and `AddSubmonoid`, wouldn't it be more coherent to have a (possibly localized, in the sense of the namespace `AddOreLocalization`) declaration `AddOreSet`?

Reply:
The current `OreSet` is under the namespace `OreLocalization`. Do you mean `AddOreLocalization.AddOreSet`?

Reply:
It seems to me that right now there are two declarations `oreSet`, one in `OreLocalization` and one in `AddOreLocalization`. I am suggesting to rename the second to become `AddOreSet`. As to whether we should keep the namespace `AddOreLocalization`, I'm happy with whatever you suggest. Or did I misunderstand your question?

Reply:
Renamed to `AddOreLocalization.AddOreSet`.